src/core/MIGRATION.md

```diff
@@ -1093,7 +1093,18 @@ _See also: [Server's CoreSetup API Docs](/docs/development/core/server/kibana-pl
 
 ### Configure plugin
 Kibana provides ConfigService if a plugin developer may want to support adjustable runtime behavior for their plugins. Access to Kibana config in New platform has been subject to significant refactoring.
-In order to have access to a config, plugin *should*:
+
+Config service does not provide access to the whole config anymore. New platform plugin cannot read configuration parameters of the core services nor other plugins directly. Use plugin contract to provide data.
+
+```js
+// your-plugin.js
+// in Legacy platform
+const basePath = config.get('server.basePath');
```

Comment:
TODO use this example instead https://github.com/restrry/kibana/blob/71119bddee66a7ec20482e4791706cd0f58e7471/src/core/server/http/http_server.ts#L231